charts/chart-colors.json

```diff
@@ -0,0 +1,278 @@
+
```

Comment:
@julio8a This doesn't appear to be a valid JSON file. Seems like it needs to be linted and valid JSON so it could be imported.